sources/platform/server/auth-source-control.md

```diff
@@ -141,10 +142,12 @@ Please use the table below to find the appropriate add-on for your BitBucket Ser
 
 ### Finish Admiral config
 
-* Switch back to Admiral and click on **Configure & Install**
-* Enter the password/secret specified in Step 4 in **Client ID**
-* Enter the private key (contents of `id_rsa` file from the **Generate SSH keys** step) in **Client Secret**
-* Please note that you will need to click on **Save** and **Restart Services** for your changes to be effective
+* Switch back to Admiral and click on **Configure & Install**.
```

Comment:
theres no `configure and install` section now. this will go in `control plane` section i suppose